src/openaq/incremental_downloader_parallel.py

```
@@ -484,7 +484,7 @@ def download_country_all(self, country_code: str, country_id: int,
                 batch_start = time.time()
                 
                 try:
-                    self.save_checkpoint(country_code, i, total_locations, completed_locations,
+                    self.save_checkpoint(country_code, start_index + i, total_locations, completed_locations,
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)